docs/SPEC.md

```
@@ -276,20 +311,51 @@ Steps and substeps may declare INPUTS and OUTPUTS directives for passing data be
 
 OUTPUTS declares values to persist after a successful step execution.
 
+*   **Scope**: Supported on both H2 steps and H3 substeps. At most one OUTPUTS directive per step or substep (Conformance §8 rule 11).
```

markdownlint-cli2 (0.22.0)

[warning] Spaces after list markers
Expected: 1; Actual: 3

(MD030, list-marker-space)